src/components/Typography/Typography.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useMemo } from 'react';
+import classNames from 'classnames';
+
+import styles from './Typography.module.css';
+
+type TagType =
```

Comment:
Would it work to use [React.ElementType]( https://flow.org/en/docs/react/types/#toc-react-elementtype) here?

Reply:
Yeah, that works. I'm cleaning up the code to use that instead. Thanks